<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartLook - Seu closet virtual!</string>
  </property>
  <property name="whatsThis">
   <string>TETSTETETE</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="QPushButton" name="btnSair">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>370</y>
        <width>200</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rage Italic</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
font: 23pt &quot;Rage Italic&quot;;</string>
      </property>
      <property name="text">
       <string>Sair</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMatch">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>200</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rage Italic</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
font: 23pt &quot;Rage Italic&quot;;</string>
      </property>
      <property name="text">
       <string>Combine!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEstatistica">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>200</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rage Italic</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
font: 23pt &quot;Rage Italic&quot;;</string>
      </property>
      <property name="text">
       <string>Estatisticas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNovo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>135</y>
        <width>200</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rage Italic</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
font: 23pt &quot;Rage Italic&quot;;</string>
      </property>
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCloset">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>200</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rage Italic</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
font: 23pt &quot;Rage Italic&quot;;</string>
      </property>
      <property name="text">
       <string>Guarda roupa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/back.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>btnSair</zorder>
     <zorder>btnMatch</zorder>
     <zorder>btnEstatistica</zorder>
     <zorder>btnNovo</zorder>
     <zorder>btnCloset</zorder>
    </widget>
    <widget class="QWidget" name="closet">
     <widget class="QLabel" name="labelTronco">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/t.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTroncoRight">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_a.png</normaloff>:/img/btn_a.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTroncoLeft">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_b.png</normaloff>:/img/btn_b.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelPernas">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>165</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/p.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelCalcado">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/c.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPernasRight">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>165</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_a.png</normaloff>:/img/btn_a.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcadoRight">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_a.png</normaloff>:/img/btn_a.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPernasLeft">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>165</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_b.png</normaloff>:/img/btn_b.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcadoLeft">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_b.png</normaloff>:/img/btn_b.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTronco">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPernas">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>165</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcado">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/wall.jpg</pixmap>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>labelTronco</zorder>
     <zorder>btnTroncoRight</zorder>
     <zorder>btnTroncoLeft</zorder>
     <zorder>labelPernas</zorder>
     <zorder>labelCalcado</zorder>
     <zorder>btnPernasRight</zorder>
     <zorder>btnCalcadoRight</zorder>
     <zorder>btnPernasLeft</zorder>
     <zorder>btnCalcadoLeft</zorder>
     <zorder>btnTronco</zorder>
     <zorder>btnPernas</zorder>
     <zorder>btnCalcado</zorder>
    </widget>
    <widget class="QWidget" name="estatistica">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>511</width>
        <height>425</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tabelaStats">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Total de peças:</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Combinações possíveis: </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Cor mais abudante:  </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Camisas, Blusas, etc: </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Calças, Bermudas e etc: </string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Tênis, chinelos e etc</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="tabela">
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Nome</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Peça</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cor</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tamanho</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="btnOrdenar">
             <item>
              <property name="text">
               <string>Ordenar por</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Peça</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="match">
     <widget class="QLabel" name="labelMatchT">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>15</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/t.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelMatchP">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>245</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/p.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelMatchC">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>245</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/c.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMatch_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>15</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rage Italic</family>
        <pointsize>23</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="scr.qrc">
        <normaloff>:/img/btn_random.png</normaloff>:/img/btn_random.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="scr.qrc">:/img/wall.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>labelMatchT</zorder>
     <zorder>labelMatchP</zorder>
     <zorder>labelMatchC</zorder>
     <zorder>btnMatch_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionCarregar"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionApagar_Tudo"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuIdioma_2">
    <property name="title">
     <string>Idioma</string>
    </property>
    <addaction name="actionpt_BR"/>
    <addaction name="actionen_US"/>
    <addaction name="actiones_ES"/>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Início</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="separator"/>
    <addaction name="actionCloset"/>
    <addaction name="actionNovo"/>
    <addaction name="actionEstatisticas"/>
    <addaction name="actionCombine"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuSobre"/>
   <addaction name="menuMenu"/>
   <addaction name="menuIdioma_2"/>
  </widget>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionApagar_Tudo">
   <property name="text">
    <string>Excluir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionpt_BR">
   <property name="text">
    <string>Português (Brasil)</string>
   </property>
  </action>
  <action name="actionen_US">
   <property name="text">
    <string>English (United States)</string>
   </property>
  </action>
  <action name="actiones_ES">
   <property name="text">
    <string>Español (España)</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionCloset">
   <property name="text">
    <string>Closet</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEstatisticas">
   <property name="text">
    <string>Estatisticas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionCombine">
   <property name="text">
    <string>Combine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="scr.qrc"/>
 </resources>
 <connections/>
</ui>
